runtime/lua/vim/lsp.lua

```diff
@@ -524,7 +524,7 @@ function lsp.start_client(config)
   function client.request(method, params, callback, bufnr)
     if not callback then
       callback = resolve_callback(method)
-        or error("not found: request callback for client "..client.name)
+        or error(string.format("not found: %q request callback for client %q.", method, client.name))
```

Comment:
What is `%q`? I couldn't find it on lua docs.

Reply:
See https://www.lua.org/manual/5.1/manual.html#pdf-string.format :)

Reply:
Excellent. ty